data/Parsifal-M.json

```
@@ -25,5 +42,15 @@
       "url": "https://hachyderm.io/@parcifal",
       "icon": "FaMastodon"
     }
+  ],
+  "milestones": [
+    {
+      "title": "Became an OPA (Open Policy Agent) community champion",
+      "date": "Dec 2022",
+      "icon": "FaGrinStars",
+      "color": "grey",
+      "description": "Became an OPA (Open Policy Agent) community champion by helping out the community and contributing to the project!",
+      "url": "https://www.openpolicyagent.org/community"
+    }
   ]
-}
+}
```

```suggestion
}

```

Missing endline may cause issues with IDEs and configurations